drivers/bridge/setup_ip_tables.go

```diff
@@ -13,31 +13,35 @@ import (
 const (
 	DockerChain    = "DOCKER"
 	IsolationChain = "DOCKER-ISOLATION"
+	ServiceChain   = "DOCKER-SERVICE"
```

Comment:
Yeah, good idea. In fact the `DOCKER-EXPOSED` filter chain is for accepting any packet for the container exposed ports before packet is being sent to the isolation chain.